configs/config.json.example

```diff
@@ -74,6 +74,7 @@
     "evolve_captured": "NONE",
     "catch_randomize_reticle_factor": 1.0,
     "catch_randomize_spin_factor": 1.0,
+    "move_to_fort_even_if_full_inventory": false,
```

Comment:
Hmm. Got it.

Perhaps the tasks don't actually need to share the configuration. SpinForts could have a `min_space_required` config that could be set to 0 and defaults to the 5 it is at now. It actually looks like you pretty much have that with the `spin_even_if_inventory_full` boolean. Although it doesn't look like that is actually used. In that case, the change to spin would be completely decoupled from moving to forts.